reference/7/Microsoft.PowerShell.Utility/Select-String.md

```diff
@@ -402,6 +402,34 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -Culture
+
+Specifies a culture to match the specified pattern. The **Culture** parameter must be used with the
+**SimpleMatch** parameter. The default behavior uses the culture of the current PowerShell runspace
+(session).
+
+The **Culture** parameter uses tab completion to scroll through the list of arguments that specify
+the available cultures. To list all available arguments, use the following command:
+
+`(Get-Command Select-String).Parameters.Culture.Attributes.ValidValues`
```

Comment:
I think it is important to add an information that it is not only culture strings like `en-US` but also special values like `Ordinal`, `Invariant` and `Current`.

Reply:
@iSazonov Can you open a new issue about this or, even better, submit a PR with the change you would like to see?